test/versioned/mongodb-esm/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mongodb-esm-tests",
+  "targets": [{"name":"mongodb","minAgentVersion":"1.32.0"}],
```

Comment:
This line being included should not break things, but we already have it over at:
https://github.com/newrelic/node-newrelic/blob/b32f7934fec5dc9e7b29dee5d1994ab180bb0c37/test/versioned/mongodb/package.json#L3

It's probably best to have it in only one of the suites, treating the ESM suite as an add-on suite.

Reply:
Agreed. This was here from before.